enumeratum-core/src/main/scala/enumeratum/EnumEntry.scala

```diff
@@ -19,7 +19,7 @@ trait EnumEntry {
     */
   def entryName: String = stableEntryName
 
-  private[this] lazy val stableEntryName: String = toString
+  private lazy val stableEntryName: String = toString
```

Comment:
Was this (and below) removals of `this` needed?

Reply:
the scala 3 build fails without removing these (deprecated feature)